applications/frontend/src/App.tsx

```diff
@@ -1,73 +1,89 @@
-import {Box} from "@mui/system";
-import {Autocomplete, Button, CircularProgress, TextField} from "@mui/material";
-import {useState} from "react";
-import Recipe, {RecipeData} from "./Components/Recipe";
-import IngredientOptions from './files/Ingredients.json'
+import { Box } from "@mui/system";
+import {
+  Autocomplete,
+  Button,
+  CircularProgress,
+  TextField,
+} from "@mui/material";
+import { useState } from "react";
+import Recipe, { RecipeData } from "./Components/Recipe";
+import IngredientOptions from "./files/Ingredients.json";
 
 function App() {
-    const [recipe, setRecipe] = useState({} as RecipeData)
-    const [ingredients, setIngredients] = useState([] as string[])
-    const [loading, setLoading] = useState(false);
+  const [recipe, setRecipe] = useState({} as RecipeData);
+  const [ingredients, setIngredients] = useState([] as string[]);
+  const [loading, setLoading] = useState(false);
 
-    function loadingIndicator() {
-        if (loading) {
-            return (
-                <Box>
-                    <CircularProgress/>
-                </Box>
-            )
-        }
+  function loadingIndicator() {
+    if (loading) {
+      return (
+        <Box>
+          <CircularProgress />
+        </Box>
+      );
     }
+  }
 
-    async function getRecipe() {
-        setLoading(true)
-        const requestBody = JSON.stringify({
-            ingredients: ingredients
-        })
-        await fetch("http://localhost:8000/keepAlive", {
-            method: "GET",
-            headers: {
-                "Content-Type": "application/json",
-            }
-        }).then(() => setLoading(false));
-        await fetch("http://localhost:8000/recipes", {
-            method: "POST",
-            headers: {
-                "Content-Type": "application/json",
-            },
-            body: requestBody,
-        }).then(response => response.json())
-            .then(data => setRecipe(data))
-            .finally(() => setLoading(false));
+  async function getRecipe() {
+    setLoading(true);
+    const requestBody = JSON.stringify({
+      ingredients: ingredients,
+    });
+    try {
+      await fetch("http://localhost:8000/checkLiveness", {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }).then(() => setLoading(false));
+    } catch (error) {
```

Comment:
did we figure out if this resulting in no loading symbol?

Reply:
After removing line 38 this was no longer an issue! 🥳